poc-07.py

```diff
@@ -91,15 +121,18 @@ class StateController(object):
         - the engine;
         - our state backend;
         - their state backend (read-only).
+        #Comment Nishant : I think State Controller is supposed to communicate with thier state backend 
+        (write - Only)
     """
 
     def __init__(self, driver):
         self.driver = driver # The driver we own.
         self.state = StateBackend() # Would be an emitter.
 
-    def update(self, theirMessages):
+    def update(self, theirMessages, theirstate):
```

Comment:
* variable name does not respect the style of the code base (camelCase).
* Is it the good time to get "theirState"?